tasks/update.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I personally would use consistent file extensions (`yml` vs `yaml`)

Reply:
right changed to yml